src/views/Settings.vue

```diff
@@ -157,6 +169,9 @@ export default defineComponent({
       currentShopifyConfig: 'user/getCurrentShopifyConfig',
     })
   },
+  mounted() {
+    this.appVersion = this.appInfo.version;
```

Comment:
We will show branch + revision if available, else tag
https://github.com/hotwax/ionic-sdk/pull/84/files#diff-817427b0a52f22c1023629bd0728cd3102a9dd01cb780ef55f79d99ec24c23e6R78

Reply:
Sure sir.